pgd_core/templates/profile.html

```diff
@@ -5,6 +5,31 @@
 
 {% block content %}
 	<head> Welcome, {{full_name}} </head>
+
+	<div style="position:fixed; top:70px; right:90px; width:200px;">
+		<form method="post">{% csrf_token %}
+
+			{{ form.query.label }} <br>
+			{{ form.query}} <br>
+			{% for choice in form.search_type %}
+	  			{{ choice.choice_label }}
+	  			<span class="radio"> {{ choice.tag }} </span> <br>
+			{% endfor %}
+
+		</form>
+	</div> 
+
+<style>
```

Comment:
Could you please elaborate this ? :) i was able to understand what you said yesterday, but not now :(